lib/ec2imgutils/ec2uploadimg.py

```diff
@@ -752,6 +752,17 @@ def _launch_helper_instance(self):
                         'SubnetId': self.vpc_subnet_id,
                         'Groups': self.security_group_ids.split(',')
                     }
+                ],
+                TagSpecifications=[
+                    {
+                        'ResourceType': 'instance',
+                        'Tags': [
+                            {
+                                'Key': 'Name',
+                                'Value': 'ec2uploadimg-helper-instance',
+                            },
```

Comment:
Trailing comma?

Reply:
From copy/paste, technically not invalid format but cleaned up anyway.